### PR TITLE
Inserting data to db as a block

### LINKED ISSUE #243: Data needs to be inserted to db as block
`Aggregator` sends data to the database by looping through a list of entries. This should be done in a single insert, so that all the new entries have the same timestamp.

### PR DESCRIPTION
**Pull request by jeandemeusy**
### Current situation
Each time `Aggregator` has to push data to the database table, multiple entries are pushed. Until now, entries were pushed one by one, inducing a different timestamp for each entry.

### What's new
This PR uses the `insert_many` method of the `DatabaseConnection` class to push multiple entries at the same time.

Resolves #243

### CODE REVIEW

File: ct-app/aggregator/routes.py [hunk 1]
```diff
@@ -153,14 +153,11 @@ async def post_to_db(request: Request):  # pragma: no cover
...
-
-            for peer, nodes, latencies in matchs_for_db:
-                log.info(f"Inserting {peer} into DB")
-                db.insert(
-                    "raw_data_table",
-                    peer_id=peer,
-                    node_addresses=nodes,
-                    latency_metric=latencies,
+            if len(matchs_for_db) > 0:
+                db.insert_many(
```
Comment (lichinka) [inline ct-app/aggregator/routes.py:157]: Make sure to capture potential exceptions raised by this code.

File: ct-app/aggregator/routes.py [hunk 2]
```diff
@@ -153,14 +153,11 @@ async def post_to_db(request: Request):  # pragma: no cover
...
+                    "foo_table",
```
Comment (lichinka) [inline ct-app/aggregator/routes.py:158]: FYI, usually table names are members of their respective data classes.

**Review by lichinka — CHANGES_REQUESTED** [review]

File: ct-app/aggregator/routes.py (hunk 2 above)
Comment (jeandemeusy) [reply ct-app/aggregator/routes.py]: Resolved in future PR: #251 (issue #248)

File: ct-app/aggregator/routes.py (hunk 1 above)
Comment (jeandemeusy) [reply ct-app/aggregator/routes.py]: Done !

**Review by MartinBenediktBusch — APPROVED** [review]

**Review by lichinka — APPROVED** [review]
Good. Let's open another issue to bubble exceptions up to the level that the application has enough context information to "correctly" react to them.

**Pull request merged**